### ISSUE TITLE
[BUG] /setup often results in PHP OOM

### ISSUE THREAD

**Issue by charles-langley** [goryn-clade/pathfinder#56]
/setup shows a PHP memory_limit requirement of 128M, but loading /setup and/or executing tasks often returns a 500 error due to "PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted". Bumping memory_limit up to 256M appears to resolve the issue.

Error:
`PHP message: PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 20480 bytes) in /path/to/pathfinder/vendor/bcosca/fatfree-core/db/sql/mapper.php on line 659`

Note: It was reported by fleischsalatinspace in Stash that this issue has been resolved for docker builds. I'm reporting as requested in order to apply the memory limit display fix to the main repo as well.

**Comment by fleischsalatinspace** [comment]
docker fix is here https://github.com/goryn-clade/pathfinder-containers/issues/2

**Comment by samoneilll** [comment]
Transferred to pathfinder-containers project.

**Comment by fleischsalatinspace** [comment]
i may be mistaken, but i think this change has to happen in the pathfinder repository, as the described issue resides in 
`app/requirements.ini`
```
; max memory limit
; some requests e.g. build indexes on /setup page require more RAM
MEMORY_LIMIT                =   128M
```
Afaik it only affects bare-metal installations

**Comment by samoneilll** [comment]
Fixed in #580172d81582044d16d6e45c28e2f299537447d6, will be included in next release
